tools/topology/sof/pipe-processing-playback.m4.c

```diff
@@ -0,0 +1,60 @@
+# Low Latency Passthrough Pipeline and PCM
+#
+# Pipeline Endpoints for connection are :-
```

Comment:
Commit message should also add BUF0 and BUF1 buffers between components. This would make the commit message more clear.

Reply:
added.